docs/_providers/axfrddns.md

```diff
@@ -0,0 +1,130 @@
+---
+name: AXFRDDNS
+title: AXFR+DDNS Provider
+layout: default
+jsId: AXFRDDNS
+---
+# AXFR+DDNS Provider
+
+This provider is able to work with any authoritative DNS server
+accepting AXFR requests (RFC5936) and Dynamic Update (RFC2136).
+
```

Comment:
I hope you don't mind some rewrites. The editor in me has some suggested edits for this doc.

Suggested replacement:

This provider uses the native DNS protocols.  It uses the
Zone Transfer Protocol (AXFR) (RFC5936) to retrieve
the existing records and  Dynamic Update (RFC2136) to make
corrections.

It is able to work with any properly configured, standards compliant,
authoritative DNS server.  It has been tested with
[BIND](https://www.isc.org/bind/),
[Knot](https://www.knot-dns.cz/), and
[Yadifa](https://www.yadifa.eu/home).

Reply:
Thanks for the suggested edits !